opendbc/car/honda/values.py

```diff
@@ -199,6 +200,12 @@ class CAR(Platforms):
     CarSpecs(mass=3338.8 * CV.LB_TO_KG, wheelbase=2.5, centerToFrontRatio=0.5, steerRatio=16.71, tireStiffnessFactor=0.82),
     {Bus.pt: 'acura_rdx_2020_can_generated'},
   )
+  HONDA_PILOT_4G = HondaBoschPlatformConfig(
+    [HondaCarDocs("Honda Pilot 2023", "All")],
+    CarSpecs(mass=4278 * CV.LB_TO_KG, wheelbase=2.86, centerToFrontRatio=0.428, steerRatio=16.0, tireStiffnessFactor=0.444),  # as spec
+    {Bus.pt: 'honda_pilot_2023_can_generated'},
+    flags=HondaFlags.CANFD | HondaFlags.BOSCH_ALT_BRAKE,
```

Comment:
referencing above, no radarless here will cause conflicts

Reply:
Honestly it's been so long since I did the initial port, so I don't remember the specifics on why this was added. I'll remove for the moment.